pkg/secrethub/configdir/dir.go

```diff
@@ -0,0 +1,71 @@
+package configdir
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+)
+
+type Dir struct {
+	Path string
+}
+
+func Default() (*Dir, error) {
+	// TODO: move to constant?
```

Comment:
As long as we use it in just one place, I'm fine with using a magical string as it actually improves readability. If we use the same thing multiple times though, we should move it to a constant.